cargo-dist/src/backend/installer/homebrew.rs

```diff
@@ -265,4 +324,40 @@ mod tests {
     fn ampersand_but_no_digit() {
         run_comparison("openssl@blah", "Openssl@blah");
     }
+
+    fn run_spdx_comparison(spdx_string: &str, homebrew_dsl: &str) {
+        let result = to_homebrew_license_format(spdx_string).unwrap();
+        assert_eq!(result, homebrew_dsl);
+    }
+
+    #[test]
+    fn spdx_single() {
```

Comment:
Thank you for including tests! Might be worth also importing Homebrew's tests wholesale so we know it works with the stuff Homebrew tests against too: https://github.com/Homebrew/brew/blob/master/Library/Homebrew/test/utils/spdx_spec.rb#L214-L239

Reply:
Added more tests to cover homebrew's cases–not all of them as this change does not support "WITH", but now parentheses and plus signs are covered, as well as some invalid cases.

Reply:
Perfect, thanks so much! Not handling `WITH` at this point makes sense given the upstream crate's limitations.